README.md

```diff
@@ -269,7 +269,6 @@ roslaunch mini_pupper_bringup bringup.launch
 #### 2.3.2 Run Cartographer
 
 **You should run this command on PC**  
-**Notice: If you are using gazebo, set the param /use_sim_time in navigate.launch as true.**
```

Comment:
if we check launch/bringup.launch, the use_sim_time is defined **inside** a `<group`, is this ok?

Reply:
Thanks for pointing it out, I'll fix it

Reply:
Fixed in [8180650](https://github.com/mangdangroboticsclub/mini_pupper_ros/pull/30/commits/8180650b8ab65c92045bafefdb113a0028c879fa)